libs/web3/src/lib/url-connector.ts

```diff
@@ -0,0 +1,170 @@
+import { ethers, Wallet } from 'ethers';
+import { Connector } from '@web3-react/types';
+import { Eip1193Bridge } from '@ethersproject/experimental';
+import type { ConnectionInfo } from '@ethersproject/web';
+import type { Actions } from '@web3-react/types';
+import { initializeConnector } from '@web3-react/core';
+
+export class CustomizedBridge extends Eip1193Bridge {
```

Comment:
chore (blocking): can we move this into it's own file, doesn't feel like it belongs here, and can we make it so token is using it as well rather than duplicating :)